### PR TITLE
SDC Resource Fixes

### PR DESCRIPTION
**Pull request by Krunal-Thakkar**
SDC Resource Fixes

image

### CODE REVIEW

**Review by shenda1 — APPROVED** [review]

**Review by AnikaAgiwal2711 — DISMISSED** [review]

File: docs/resources/sdc.md [hunk 1]
```diff
@@ -119,25 +119,25 @@ resource "powerflex_sdc" "importing" {
 
 ### Optional
 
-- `id` (String) ID of the SDC to manage. This can be retrieved from the Datasource and PowerFlex Server. Cannot be updated.
+- `id` (String) ID of the SDC to manage. This can be retrieved from the Datasource and PowerFlex Server. Cannot be updated. Conflict with `ip`
```
Comment (rounak-adhikary) [inline docs/resources/sdc.md:122]: I dont see an `ip` field. Maybe this is a silly mistake?

**Review by shenda1 — DISMISSED** [review]

File: docs/resources/sdc.md (hunk 1 above)
Comment (Krunal-Thakkar) [reply docs/resources/sdc.md:122]: It is available in the schema.

File: powerflex/sdc_resource_schema.go [hunk 2]
```diff
@@ -105,11 +105,13 @@ var SDCReourceSchema schema.Schema = schema.Schema{
 	Attributes: map[string]schema.Attribute{
 		"sdc_details": sdcDetailSchema,
 		"name": schema.StringAttribute{
+			DeprecationMessage:  "This attribute will be remove in future release. To rename SDC, use attribute `name` in `sdc_details`",
```
Comment (shenda1) [inline powerflex/sdc_resource_schema.go:108]: will be removed?

Comment (Krunal-Thakkar) [reply powerflex/sdc_resource_schema.go]: Updated.

File: docs/resources/sdc.md (hunk 1 above)
Comment (Krunal-Thakkar) [reply docs/resources/sdc.md]: Updated.

**Review by shenda1 — DISMISSED** [review]

**Review by rounak-adhikary — APPROVED** [review]
lgtm

**Pull request merged**